lib/threefry.ex

```diff
@@ -0,0 +1,90 @@
+defmodule PRNG.Threefry do
+  import Kernel, except: [unless: 2]
+  use Bitwise
+
+  defp is_threefry_key(key) when is_list(key) do
+    length(key) == 2 and Enum.all?(
+      fn x -> is_integer(x) end
+    )
+  end

```

Comment:
I thing it'd be nice to have it as guard:
```suggestion
  defguard is_threefry_key(term)
           when is_list(term) and length(term) == 2 and is_integer(hd(term)) and
                  is_integer(hd(tl(term)))
```